Remotion/ObjectBinding/Web/UI/Controls/BusinessObjectBoundWebControl.cs

```diff
@@ -432,10 +432,10 @@ bool ISmartControl.IsRequired
       get { return false; }
     }
 
-    IEnumerable<BaseValidator> ISmartControl.CreateValidators ()
-    {
-      return Enumerable.Empty<BaseValidator>();
-    }
+    IEnumerable<BaseValidator> ISmartControl.CreateValidators () => CreateValidators ();
+
+    /// <inheritdoc cref="ISmartControl.CreateValidators"/>
+    protected virtual IEnumerable<BaseValidator> CreateValidators () => Enumerable.Empty<BaseValidator>();
```

Comment:
We need to keep CreateValidators as seen from BusinessObjectBoundEditableWebControl as a public member. Otherwise, we introduce a breaking change for projects that access the CreateValidators() method.

=> unify the explcit interface implementation and the protected virtual with a public implicit interface impelmetnation on BusinessObjectBoundWebControl.